codestodians/food-basket-api
# [Chore: #154603149] Update Payment schema test specs

#### What does this PR do?
- Restructures the naming of test specs in the `Payment` schema

#### Description of Task to be completed?
- Rename tests to conform with the `three-digits-number-after-first-underscore` format.
- Update `Payment` schema snapshots

#### How should this be manually tested?
- Check out into the `ch-refactor-payment-specs-naming-154603149` branch.
- Run tests and check the `Payment` snapshots module. The tests should be seen arranged in a systematic order.

#### Any background context you want to provide?
- Going forward, this will make it easier to add additional tests without being worried about the alphabetical sorting of new tests written.

#### What are the relevant pivotal tracker stories?
- [#154603149](https://www.pivotaltracker.com/story/show/154603149)
